### PR TITLE
Ionian Documentation

### PR DESCRIPTION
**Pull request by ckti**
Documentation for vdi, usb and ISO Ionian images
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/cevap%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/24996551%3Fv%3D4%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cevap/ion/pull/22%23issuecomment-378001023%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222018-04-02T18%3A22%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars3.githubusercontent.com/u/24996551%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cevap%22%7D%7D%2C%20%7B%22body%22%3A%20%22have%20update%20the%20docs%20in%20pull%20request%2024%2C%20this%20one%20can%20be%20closed%22%2C%20%22created_at%22%3A%20%222018-04-03T15%3A10%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/7046769%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ckti%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2035793c643d3a4c693adb5a6433005d5aa41fe45c%20doc/ionian-doc/ionian.doc.md%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/cevap/ion/pull/22%23discussion_r178688899%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Could%20we%20please%20add%20a%20description%20for%20what%20Ionian%20is%3F%20Could%20we%20also%20please%20move%20the%20doc%20out%20to%20the%20root%20of%20the%20%60docs/%60%20folder%20and%20just%20leave%20the%20images%20in%20this%20sub%20directory%3F%22%2C%20%22created_at%22%3A%20%222018-04-03T00%3A55%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/8009243%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/mitchellcash%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ok%20will%20do%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222018-04-03T09%3A37%3A00Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/7046769%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ckti%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20doc/ionian-doc/ionian.doc.md%3AL1-76%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/cevap/ion/pull/22%23issuecomment-378001023%22%2C%20%22https%3A//github.com/cevap/ion/pull/22%23discussion_r178688899%22%2C%20%22https%3A//github.com/cevap/ion/pull/22%23discussion_r178765695%22%2C%20%22https%3A//github.com/cevap/ion/pull/22%23issuecomment-378284906%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/cevap'></a>

- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/cevap/ion/pull/22#issuecomment-378001023'>General Comment</a></b>
- <a href='https://github.com/cevap'></a> :shipit:
- <a href='https://github.com/ckti'></a> have update the docs in pull request 24, this one can be closed
- [ ] <a href='#crh-comment-Pull 35793c643d3a4c693adb5a6433005d5aa41fe45c doc/ionian-doc/ionian.doc.md 3'></a> &nbsp;<b><a href='https://github.com/cevap/ion/pull/22#discussion_r178688899'>File: doc/ionian-doc/ionian.doc.md:L1-76</a></b>
- <a href='https://github.com/mitchellcash'></a> Could we please add a description for what Ionian is? Could we also please move the doc out to the root of the `docs/` folder and just leave the images in this sub directory?
- <a href='https://github.com/ckti'></a> Ok will do

<a href='https://www.codereviewhub.com/cevap/ion/pull/22?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cevap/ion/pull/22?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/cevap/ion/pull/22?approve=0'></a>&nbsp;<a href='https://github.com/cevap/ion/pull/22'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by cevap** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/cevap/ion/pull/22'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by cevap** [comment]
:shipit:

File: doc/ionian-doc/ionian.doc.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+Ionian OS Installation Manual
+=============================
+Virtualbox VDI installation
```
Comment (mitchellcash) [inline doc/ionian-doc/ionian.doc.md]: Could we please add a description for what Ionian is? Could we also please move the doc out to the root of the `docs/` folder and just leave the images in this sub directory?

Comment (ckti) [reply doc/ionian-doc/ionian.doc.md]: Ok will do

**Comment by ckti** [comment]
have update the docs in pull request 24, this one can be closed

**Comment by ckti** [comment]
New pull request sent, this can be closed now

**Pull request closed without merging**